.github/workflows/eas-build.yml

```diff
@@ -0,0 +1,66 @@
+name: EAS PIPELINE
+on:
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: build_app
+jobs:
+  merge:
+    name: Merge main into BUILD
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Branch
+        uses: actions/checkout@v2
+
+      - uses: morbalint/git-merge-action@v1
+        with:
+          target: "BUILD"
+          strategy_options: "ours"
+
+  build:
+    name: Build and Submit
+    runs-on: ubuntu-latest
+    needs: merge
+    steps:
+      - name: Checkout Branch
+        uses: actions/checkout@v2
+        with:
+          ref: "BUILD"
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.14.2
```

Comment:
The version can be pulled from package.json so it's not specific twice. Refer to the other GHA for steps

Reply:
I tried setting it in package.json but it wasn't working so that's why I specifically installed it here

Reply:
Let's discuss. Would rather fix your problem than regress from the standards